README.md

```diff
@@ -71,6 +71,9 @@ OPTIONS:
 
 The `ragenix` package also provides shell completions for `bash`, `zsh`, and `fish`. Make sure to install the package with either `nix profile install github:yaxitech/ragenix`, `environment.systemPackages` on NixOS or `home.packages` for home-manager.
 
+## Plugin Support (i.e. YubiKey)
+With the release of [`rage`](https://github.com/str4d/rage) v0.6.0 a plugin system was introduced. Using this plugin system rage supports identities and recipients from third-parties such as the [`age-plugin-yubikey`](https://github.com/str4d/age-plugin-yubikey). Ragenix also supports this. Make sure to make the plugin available in `PATH` so that `ragenix` can find it.
```

Comment:
```suggestion

With the release of [`rage`](https://github.com/str4d/rage) v0.6.0 a plugin system was introduced. Using this plugin system `rage` supports identities and recipients from third-parties such as the [`age-plugin-yubikey`](https://github.com/str4d/age-plugin-yubikey). `ragenix` also supports this. Make sure to make the plugin available in `PATH` so that `ragenix` can find it.
```